### DISCUSSION TITLE
Invalid URL on main hub page

### DISCUSSION THREAD

**Discussion by miik-miik** [mozilla/hubs#3340] [Hubs Cloud Questions] [answered]
Hello Hello,

I have a hubs cloud on AWS and my AddressForRootDomain point direct to "invalid url" - all other sites are working, like spoke, admin panel etc. I use the hubs code base, no extra code or something like this.

thanks and greets, 
miik

**Comment by robinkwilson** [comment, marked as answer]
Hi miik, I need a little more information from you. 

Are you following Recipe 3? If you have, have you done the DNS after setup step? 
Does your stack outputs in AWS say "invalid url"?

**Reply by robinkwilson** [reply]
Marking as answered for now, when you have more details please click "unmark as answered" button!
